flocker/node/functional/test_docker.py

```diff
@@ -356,6 +398,7 @@ def image_built(image_name):
                 unit_name=unit_name,
                 image_name=image_name,
                 environment=Environment(variables=expected_variables),
+                force=True,
```

Comment:
If the only real change in behavior is in the test suite then, hopefully, our test suite will stop failing spuriously with this problem.  What about the actual implementation, though?  If this failure can strike test code that's removing a container, it seems plausible it could strike `StopApplication` when in use on a real cluster.

Is there a reason to think this isn't the case?  Or does this problem eventually solve itself so that, as long as you're retrying (as the looping approach of the convergence agent will do) the removal does eventually succeed?